src/librbd/plugin/Types.h

```diff
@@ -18,7 +17,7 @@ struct HookPoints {
 
 template <typename ImageCtxT>
 struct Interface : public ceph::Plugin {
-  Interface(CephContext* cct) : Plugin(cct) {
+  Interface(ceph::common::CephContext* cct) : Plugin(cct) {
```

Comment:
Hmm -- nowhere else in `librbd` requires this namespace scoping for `CephCommon` so I'd rather fix the root cause. It's technically pulling in a definition for `CephContext` from `common/PluginRegistry.h` which is getting it from a new `include/common_fwd.h` which we should just include in this file.

Reply:
@dillaman 
I'll give that a shot, and fix if it works.